benefits/static/css/admin/styles.css

```diff
@@ -90,9 +102,89 @@ html[data-theme="light"],
   row-gap: 1rem;
 }
 
-/* Enrollment */
+/* Enrollment Page*/
 iframe.card-collection {
   border: 0px;
   width: 100%;
   height: 60vh;
 }
+
+/* Login Page */
```

Comment:
This is copied + pasted from https://github.com/django/django/blob/9ca1f6eff6f19d1ae074d289c6c4209073351805/django/contrib/admin/static/admin/css/login.css#L1 except with a few modifications necessary for the design of our page.